src/pages/swap/index.tsx

```diff
@@ -264,7 +264,7 @@ export const SwapPage = () => {
         <UserBalance token={fromToken} onClick={(amount: string) => form.setValue('fromAmount', amount)} />
       </>
     ),
-    [form, fromToken, setModalType],
+    [form, fromToken, setModalType, trackEvent],
```

Comment:
Why do we need to add `trackEvent` here?

Reply:
Because `trackEvent` is called above. After merging the latest changes into this branch it's not part of the PR anymore though.